app/pages/clinicworkspace/ClinicPatients.js

```diff
@@ -144,80 +260,370 @@ export const ClinicPatients = (props) => {
+                    {t('% Time in Range')}
+                    {!!activeFilters.timeInRange.length && (
+                      <Pill
+                        label="filter count"
+                        round
+                        width="14px"
+                        fontSize="9px"
+                        lineHeight="15px"
+                        ml={1}
+                        sx={{
+                          textAlign: 'center',
+                          display: 'inline-block',
+                        }}
+                        colorPalette={['purpleMedium', 'white']}
+                        text={`${activeFilters.timeInRange.length}`}
+                      />
+                    )}
+                  </Button>
+
+                  {activeFiltersCount > 0 && (
+                    <Button
+                      id="profileEditButton"
+                      variant="textSecondary"
+                      onClick={handleResetFilters}
+                      fontSize={0}
+                      color="grays.4"
+                    >
+                      {t('Reset Filters')}
+                    </Button>
+                  )}
+                </>
+              )}
+            </Flex>
+
+            <Flex
+              alignItems="center"
+              justifyContent="flex-end"
+            >
+              {showSummaryData && showNames && (
+                <Flex pr={3} py={1} mr={2} alignItems="center" sx={{ borderRight: borders.divider }}>
+                  <Icon
+                    mr={2}
+                    id="refresh-patients"
+                    variant="default"
+                    icon={RefreshRoundedIcon}
+                    color={loading ? 'text.primaryDisabled' : 'inherit'}
+                    disabled={loading}
+                    label={t('Refresh patients list')}
+                    onClick={handleRefreshPatients}
+                  />
+
+                  <Text fontSize={0}>{timeAgoMessage}</Text>
+                </Flex>
+              )}
+
+              <Icon
+                id="patients-view-toggle"
+                variant="default"
+                color="grays.4"
+                ml={1}
+                icon={VisibilityIcon}
+                label={t('Toggle visibility')}
+                onClick={handleToggleShowNames}
+              />
+            </Flex>
           </Flex>
-
-          <Button
-            id="add-patient"
-            variant="primary"
-            onClick={handleAddPatient}
-            mr={0}
-          >
-            {t('Add a New Patient')}
-          </Button>
         </Flex>
-      </Flex>
+      </>
     );
   };
 
+  function handleRefreshPatients() {
+    trackMetric(prefixPopHealthMetric('Refresh data'), { clinicId: selectedClinicId });
+    let fetchOptions = { ...patientFetchOptions };
+    if (isEmpty(fetchOptions.search)) delete fetchOptions.search;
+    dispatch(actions.async.fetchPatientsForClinic.bind(null, api, clinic.id, fetchOptions)());
```

Comment:
I don't _think_ this `.bind` is necessary

Reply:
Correct.  Fixed this (and one like it)